### PR TITLE
Feature/add read only user to database

### PR DESCRIPTION
**Pull request by paulcoxsnh**

### CODE REVIEW

File: src/config/database.js [hunk 1]
```diff
@@ -19,6 +19,15 @@ if (process.env.NODE_ENV === 'production') {
...
       schema: 'sfo',
       logging: false
+    },
+    ssDatabase: {
+      username: 'rosfo',
+      password: config.roSfoPassword,
+      database: 'licensing',
+      host: config.databaseHost,
+      dialect: 'postgres',
+      schema: 'traps',
```
Comment (JohnJAMoore) [inline src/config/database.js:29]: should this not be sfo?

**Review by JohnJAMoore — CHANGES_REQUESTED** [review]
Just one little thing

File: src/config/database.js (hunk 1 above)
Comment (paulcoxsnh) [reply src/config/database.js:29]: Totally - nice spot.

**Review by JohnJAMoore — APPROVED** [review]
LGTM 👍

**Pull request merged**